attributes/default.rb

```diff
@@ -81,12 +81,22 @@
 
 default['haproxy']['install_method'] = 'package'
 
+# if source determine target
```

Comment:
We should probably move this logic to a library vs. an attribute, but this works for now.